examples/plug-and-play/demo_PnP_mirror_descent.py

```diff
@@ -0,0 +1,132 @@
+r"""
+Plug-and-Play algorithm with Mirror Descent for Poisson noise inverse problems.
+====================================================================================================
+
+This is a simple example to show how to use a mirror descent algorithm for solving an inverse problem with Poisson noise.
+
+The Mirror descent with RED denoiser writes
+
+.. math:: 
+
+    x_{k+1} = \nabla \phi ( \nabla \phi^*(x_k) - \tau \nabla \distance{A(x_k)}{y} - \tau ( x_k - D_\sigma(x)))
+
+where :math:`\phi` is a convex Bergman potential, :math:`\distance{A(x)}{y}` is the data fidelity term and :math:`D_\sigma(x)` is a denoiser.
+
+In this example, we use the DnCNN denoiser. As the observation has been corrupted with Poisson noise, we use the :class:`deepinv.optim.PoissonLikelihood` data-fidelity term.
+In https://publications.ut-capitole.fr/id/eprint/25852/1/25852.pdf, it is shown that, with this data-fidelity term, the right Bregman potential to use is Burg's entropy :class:`deepinv.optim.BurgEntropy`.
```

Comment:
for some reason the link to BurgEntropy does not work here

Reply:
if you do :meth:`deepinv.optim.BurgEntropy` ?

Reply:
fixed it with :class:`deepinv.optim.bregman.BurgEntropy`!